### PR TITLE
Create netlify-access.md

### PR DESCRIPTION
**Pull request by divya-mohan0209**
This PR has been opened for documenting SIG Docs contributors who have elevated Netlify access per [issue #38839](https://github.com/kubernetes/website/issues/38839).

This is to ensure : 
- we do not give more access than is necessary and,
- we are able to correctly onboard & offboard relevant folks.

This eliminates the necessity of opening up clean-up PRs. 

A good follow-up would be to document this as part of the onboarding/offboarding checklist.

### CODE REVIEW

**Comment by divya-mohan0209** [comment]
@natalisucks @sftim @tengqm @onlydole : Would appreciate your comments on this, please!

**Comment by cblecker** [comment]
Do we/should we align this with the GitHub admin subproject that handles some Netlify access too?

**Comment by divya-mohan0209** [comment]
@cblecker That was the original idea of the initiative and I had raised an issue reg documenting the current accesses and then streamlining as a subsequent measure in this repo. I also flagged it in #sig-contribex. However, I didn't receive any specific responses which is why I documented one that catered to SIG Docs only.

**Comment by divya-mohan0209** [comment]
Issue #7311 is the one raised under kubernetes/community.

**Comment by cblecker** [comment]
I think we can probably just say that the [GH admin team](https://github.com/kubernetes/community/tree/master/github-management#github-administration-team) should also be owners and pull back all other access.

Thoughts @kubernetes/owners ?

**Comment by cblecker** [comment]
(Sorry that we didn't see/respond to this sooner -- I didn't see the original issue/pings)

**Comment by mrbobbytables** [comment]
> I think we can probably just say that the [GH admin team](https://github.com/kubernetes/community/tree/master/github-management#github-administration-team) should also be owners and pull back all other access.
> 
> Thoughts @kubernetes/owners ?

+1

**Comment by BenTheElder** [comment]
> I think we can probably just say that the [GH admin team](https://github.com/kubernetes/community/tree/master/github-management#github-administration-team) should also be owners and pull back all other access.

Not thrilled about this as a maintainer of a subproject docs site even though I see currently this only seems to discuss k/website.

In my experience there are important, technical settings that cannot be managed by in-repo config and are preferably managed by someone maintaining the site sources, such as the build image (operating system/version in which the build occurs).

I think SIG Docs folks (and the individual site maintainers more generally) are best equipped to toggle these settings.

**Comment by cblecker** [comment]
@BenTheElder Sorry, my comment about GH admins was in addition to the sig docs folks (but I can see how that wasn't clear).

What settings do you, as a subproject docs site maintainer, need to toggle in the portal as opposed to in you repo configs?

**Comment by BenTheElder** [comment]
> @BenTheElder Sorry, my comment about GH admins was in addition to the sig docs folks (but I can see how that wasn't clear).

D'oh, yes that wasn't clear to me. That makes _much_ more sense. Sorry.

> What settings do you, as a subproject docs site maintainer, need to toggle in the portal as opposed to in you repo configs?

Off the top of my head the long standing biggest problem has been the build environment. You can control a lot of settings in-repo but switching build VM? container? images can break everything and is only controlled in the web UI unfortunately, sometimes it's just fine and sftim has toggled it for us but with more bandwidth it would be better to plan it and be prepared to rollback etc.

Eventually you _have_ to do this, because whatever image you're using gets too old / deprecated. YMMV on frequency.

**Comment by MadhavJivrajani** [comment]
>  GH admins was in addition to the sig docs folks (but I can see how that wasn't clear)

+1

**Comment by natalisucks** [comment]
I like the addition of GH admins plus Docs leads as a sort of "break glass in case of emergency" aspect, so +1.

@divya-mohan0209 I remember we spoke about a designated staff member of the CNCF also having some kind of elevated permissions. WDYT? Happy to hear thoughts from others here.

File: sig-docs/netlify-access.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+- Rey Lejano (@reylejano)
+
+## Collaborator access
+
+- Karen Bradshaw (@kbhawkey)
+- Nate Waddington (@nate-double-u)
+- Patrice Chalin (@chalin)
+- Qiming Teng (tengqm)
+- Taylor Dolezal (@onlydole)
+- Tim Bannister (@sftim)
```
Comment (MadhavJivrajani) [inline sig-docs/netlify-access.md:24]:
```suggestion
- Karen Bradshaw ([@kbhawkey](https://github.com/kbhawkey))
- Nate Waddington ([@nate-double-u](https://github.com/nate-double-u))
- Patrice Chalin ([@chalin](https://github.com/chalin))
- Qiming Teng ([@tengqm](https://github.com/tengqm))
- Taylor Dolezal ([@onlydole](https://github.com/onlydole))
- Tim Bannister ([@sftim](https://github.com/sftim))
```

**Review by MadhavJivrajani — COMMENTED** [review]
Small suggestion.

**Comment by palnabarun** [comment]
> I think we can probably just say that the [GH admin team](https://github.com/kubernetes/community/tree/master/github-management#github-administration-team) should also be owners [...]

+1 to keep access to @kubernetes/owners

Also, echoing my comment from the parent issue: https://github.com/kubernetes/website/issues/38839#issuecomment-1375340379

**Comment by divya-mohan0209** [comment]
> 

CNCF folks, Nate and Patrice, have also been added to this list. I believe they were working with Bob to get elevated access to all of the other K8s subproject docs, as well.

+1 to the suggestion of GitHub admins having elevated privileges. Currently, we also have a few other folks on Netlify with elevated accesses such as previous and current Steering members, subproject folks etc. As one of the next steps, we might want to set up a cadence to periodically review if all of those folks need to be on there.

**Comment by Priyankasaggu11929** [comment]
> I like the addition of GH admins plus Docs leads as a sort of "break glass in case of emergency" aspect

+1 

> Currently, we also have a few other folks on Netlify with elevated accesses such as previous and current Steering members, subproject folks etc. As one of the next steps, we might want to set up a cadence to periodically review if all of those folks need to be on there.

Agree -- to have a documented cadence for reviewing the owners list periodically.

**Comment by divya-mohan0209** [comment]
@MadhavJivrajani: I have added the GitHub handles like you suggested.

Rest of the folks - I've mentioned the bit regarding the GitHub Admins also having access. PTAL and advise if this is good to merge.

**Comment by divya-mohan0209** [comment]
Folks - gentle reminder on this one.

File: sig-docs/netlify-access.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+This document lists SIG Docs contributors with elevated Netlify access to maintain the Kubernetes documentation.
+
+The members listed below are, typically, co-chairs or leads.
+
+The list will need to be updated during the onboarding and offboarding from these roles.
+
+In addition to the folks below, members of the [GitHub Administration team](https://github.com/kubernetes/community/tree/master/github-management#github-administration-team) also have elevated access. 
+
+To learn more about the difference between the two roles, please refer to [Netlify's official docs](https://docs.netlify.com/accounts-and-billing/team-management/team-member-roles/#collaborators).
```
Comment (kbhawkey) [inline sig-docs/netlify-access.md:11]: nit: you can omit,  'please'
nit: Are the roles 'Owner' and 'Collaborator'?

**Review by sftim — COMMENTED** [review]
/lgtm

Thanks for the reminder.

**Pull request merged**